modules/client/src/tokenVoting/internal/utils.ts

```diff
@@ -128,6 +128,16 @@ export function toTokenVotingProposalListItem(
       address: proposal.dao.id,
       name: proposal.dao.subdomain,
     },
+    settings:{
+      supportThreshold: decodeRatio(BigInt(proposal.supportThreshold), 6),
+      duration: parseInt(proposal.endDate) -
+        parseInt(proposal.startDate),
+      minParticipation: decodeRatio(
+        (BigInt(proposal.minVotingPower) * BigInt(1000000)) /
```

Comment:
Isn't minVotingPower already the token supply that needs to vote?

Reply:
Yes, but is used to compute the minParticipation in the same way we receive it betweeen 0 and 1